test/simulator/moves/snarl.js

```diff
@@ -31,9 +32,9 @@ describe('Snarl [Gen 5]', function () {
 			[{species: "Deoxys-Attack", ability: 'victorystar', item: 'laggingtail', moves: ['splash', 'snarl']}],
 			[{species: "Caterpie", level: 2, ability: 'naturalcure', item: 'focussash', moves: ['substitute', 'rest']}],
 		]);
-		battle.commitDecisions();
-		battle.choose('p1', 'move 2');
-		battle.choose('p2', 'move 2');
+
+		battle.makeChoices('move Snarl', '');
```

Comment:
This part looks wrong.

It should be:

```
battle.makeChoices('move Splash', 'move Substitute');
battle.makeChoices('move Snarl', 'move Rest');
```

Reply:
Remember, when refactoring old code:

`battle.commitDecisions` means "continue onto the next turn, making the default choice for any choices that still need to be made"

`battle.choose` means "make the choice, then, if all choices have been made, continue onto the next turn"